frontend/src/pages/Room/components/GameTimer/index.tsx

```diff
@@ -1,5 +1,8 @@
 import { Box, Text } from '@chakra-ui/react';
 import { useEffect, useState } from 'react';
+import useSound from 'use-sound';
+
+import tickTock from '../../../../assets/sound/tick-tock.mp3';
```

Comment:
alias되어있어서 요렇게 경로로 변경해도 좋을것 같아요~

```suggestion
import tickTock from 'assets/sound/tick-tock.mp3';
```